### DISCUSSION TITLE
Unicode bidi attacks

### DISCUSSION THREAD

**Discussion by cobbdb** [rehypejs/rehype#80] [Q&A] [answered]
Hello, does rehype-sanitize employ measures against unicode bidi attacks?

https://trojansource.codes/
https://github.com/lirantal/anti-trojan-source

Also asked in react-markdown: https://github.com/remarkjs/remark/discussions/904

**Comment by wooorm** [comment, marked as answer]
Unicode bidi attacks do not mean that bidi is dangerous in an of itself. There‘s nothing an HTML sanitizer can do about it. unicode bidi is fine and good. The problem is in code editors. Stuff like VSCode and GitHub. Not here.
